.gitignore

```diff
@@ -1,5 +1,11 @@
 .benchmarks.done
 .browsers
+__pycache__
 PUBLIC
 PRIVATE
-demos
\ No newline at end of file
+add_demo_src/__pycache__
+add_demo_src/tests/__pycache__
```

Comment:
The line `__pycache__` will take care of these lines 6 and 7, there is no need to specify `subdir/__pycache__` again.